Sensor sampler pm (#5753)
nrfconnect/sdk-nrf: doc/nrf/libraries/caf/sensor_sampler.rst

   * :c:member:`sensor_config.chan_cnt` - Size of the :c:member:`sensor_config.chans` array.
   * :c:member:`sensor_config.sampling_period_ms` - Sensor sampling period, in milliseconds.
   * :c:member:`sensor_config.active_events_limit` - Maximum number of unprocessed :c:struct:`sensor_event`.

>    * :c:member:`sensor_config.suspend_pm_state` - The power state to set when suspend event is received.

Author: ok

>    * :c:member:`sensor_config.suspend_pm_state` - The power state to set for given sensor when suspend event is received.

Reviewer: ```suggestion
   * :c:member:`sensor_config.suspend_pm_state` - The power state to be set for the sensor when it receives a suspend event.
```
